lib/libutee/include/tee_api_defines_extensions.h

```diff
@@ -1,12 +1,22 @@
 /* SPDX-License-Identifier: BSD-2-Clause */
 /*
- * Copyright (c) 2014, Linaro Limited
+ * Copyright (c) 2014-2021, Linaro Limited
  * Copyright (c) 2021, SumUp Services GmbH
  */
 
 #ifndef TEE_API_DEFINES_EXTENSIONS_H
 #define TEE_API_DEFINES_EXTENSIONS_H
 
+/*
+ * API extended result codes as per TEE_Result IDs defined in GPD TEE
+ * Internal Core API specification v1.1:
+ *
+ * 0x70000000 - 0x7FFFFFFF: Reserved for implementation-specific return
+ *			    code providing non-error information
+ * 0x80000000 - 0x8FFFFFFF: Reserved for implementation-specific errors
+ */
+#define TEE_ERROR_DEFER_DRIVER_INIT	0x70000000
```

Comment:
It no big deal, but in the commit message you describe this as an indication of a failure, but you have selected a value in the non-error range.

Reply:
Right. From previous discussion, we said the code reported a error case, so I'll change to `0x80000000`.